src/resources/js/views/datepicker.js

```diff
@@ -642,6 +642,16 @@ tribe.events.views.datepicker = {};
 		obj.options.beforeShowMonth = obj.filterMonthCells;
 		obj.options.beforeShowYear = obj.filterYearCells;
 
+		// A tweak for RTL languages.
+		if ( document.dir === 'rtl' ) {
+			obj.options.rtl = true;
+		}
+
+		// ...and for document language.
+		if ( document.lang ) {
```

Comment:
Check if this is defined, just in case.

Reply:
That's all this is doing 😉 - if it exists, we set it.